modules/create-policy-from-system-policies-view.adoc

```diff
@@ -43,12 +43,21 @@ However, you cannot use regular expressions for selecting deployments.
 The *Excluded Images* setting only applies when you check images in a continuous integration system with the *Build* lifecycle stage.
 It will not have any effect if you use this policy to check running deployments in the *Deploy* lifecycle stage or runtime activities in the *Runtime* lifecycle stage.
 ====
+. Optional: Under the *MITRE ATT&CK* section, select the *Tactics* and the *Techniques* you want to specify for the policy.
+.. Click *Add tactics* and then select a tactic from the dropdown list.
+.. Click the add icon to add *techniques* for the selected tactic. You can specify multiple techniques for a tactic.
```

Comment:
I believe the icon name, as a UI element, should be in bold. What do you think?

So: `the *add* icon` or, perhaps, `the *Add* icon`, depending on the UI hover help.

Reply:
Consider changing `*techniques*` to just plan `techniques`. Here, it feels more like a general description (as it is later in this sentence) than a specific UI element. But without seeing this screen, I can't be sure.

Reply:
+1 to no bold of "techniques" in line 48.